### ISSUE TITLE
ANC Subs Visit - Next Steps - Gonorrhea [2h]

### ISSUE THREAD

**Issue by jeanette-pelizzon** [TIP-Global-Health/eheza-app#245]
## Issue 

When a patient identifies that they have the following symptoms, we provide health education and medication: 
(1)Symptom Review: Abnormal Vaginal Discharge + partner has urethral discharge (follow-up question)

This diagnosis is unique as we are also recommending medication for the patient's partner. We are not following up on the dispensing of this medicine but do want to have the instructions on the screen for the health care worker to see. 

Pop-Up | Health Education | Medication
--|--|-- 
<img width="304" alt="Screen Shot 2022-04-26 at 5 37 57 PM" src="https://user-images.githubusercontent.com/77505464/165396764-7762e46a-60c3-4c7a-ba10-f9f9d777e11c.png"> | <img width="244" alt="Screen Shot 2022-05-11 at 12 48 39 PM" src="https://user-images.githubusercontent.com/77505464/167904136-b6d2ca17-cd52-4fbb-9f5d-c276a620b942.png"> | <img width="187" alt="Screen Shot 2022-05-11 at 11 14 21 AM" src="https://user-images.githubusercontent.com/77505464/167885692-e3477305-dddf-4a76-b982-c9428fd54def.png">

## Supporting Documents: 
You can see the logic for next steps[ here. ](https://docs.google.com/spreadsheets/d/1awAwPTMxRMhb2CSP3ocJvbOJravIaoPb2Q-RERe9KQ8/edit#gid=1762936493)

## Notes: 
Similar to the first ANC encounter, if a patient is receiving health education for multiple things they will stack in one health education sub-activity. If they are being referred to the hospital for multiple things, there is only one referral screen. 

## To Do:  
- Style next steps screens per design 
- Adjust logic per document

**Comment by jeanette-pelizzon** [comment]
updated health education screen to include question: 

<img width="326" alt="Screen Shot 2022-05-03 at 1 04 52 PM" src="https://user-images.githubusercontent.com/77505464/166503576-1570ec52-9dbd-4bbc-9fb1-a06d9862c0f7.png">

**Comment by anvmn** [comment]
@jeanette-pelizzon 

1. At medication distribution form, we have separate section for each medication. For example:
![Selection_730](https://user-images.githubusercontent.com/17882245/167710168-5b0fac50-cbe5-4647-8e08-3b18da4ca572.png)
Please adjust the design.

2. I believe Health education header should not be saying 'burning on urination', but something else

**Comment by jeanette-pelizzon** [comment]
@anvmn 

(1) updated main issue image 
(2) burning on urination is the symptom that they are providing health education on to better/prevent - since it appears in three different ones then you can just use the same screen?

**Comment by anvmn** [comment]
@jeanette-pelizzon

But this issue symptom is abnormal vaginal discharge, not burning on urination.

**Comment by jeanette-pelizzon** [comment]
Updated.

**Comment by balagan73** [comment]
https://nutrition-ihangane.pantheonsite.io/

### QA Review

### Found issues and questions:

1. Not related to this issue, but I found a typo when clicking around:
![typo](https://user-images.githubusercontent.com/3536805/175760966-ce3f2894-eb94-4cc5-89b9-2581f5dcd411.png)

2. So far I have found no way to indicate some diseases, like gonorrhea or candidiasis at first encounter. I need to finish first encounter, come back, and only then will be the symptom review icon available, where I can specify the symptoms. Is there a way to get these diagnoses at first encounter? What are the preconditions to display symptom review? Panel says patient has noted symptoms that require follow up question. I only noted pregnancy.

![8-symptom-review](https://user-images.githubusercontent.com/3536805/175761084-0820b859-dd77-4be1-8754-60852368a969.png)

3. Currently I cannot reproduce it, but when at subsequent encounter I started with filling out the symptom review I ended up with Next steps not displaying. Just as I was about to finish the encounter, I went back and changed the symptoms, and from that time I am getting the next steps icon.

Symptom1|Symptom2|Result
-|-|-
![9-vaginal-discharge](https://user-images.githubusercontent.com/3536805/175761602-211748bf-3503-4146-b218-a771a36d9dd5.png)|![10-urethral-discharge](https://user-images.githubusercontent.com/3536805/175761605-a8fa27e3-86fd-4e24-b321-716694fc97a5.png)|![11-no-next-steps](https://user-images.githubusercontent.com/3536805/175761608-e09015b4-3337-409e-b1e0-4f714e8bf151.png)

4. The Warning popup about gonorrhea is not as descriptive, as in the issue description.
![12-gonorrhea-warning](https://user-images.githubusercontent.com/3536805/175762065-24589485-06eb-4b6f-800f-92ed1937586e.png)

5. On the health education panel Council should be Counsel. Parctices should be Practices.
![13-health-education](https://user-images.githubusercontent.com/3536805/175762082-bf652f52-b4fc-4b42-9868-6e44214cb93d.png)

6. On the medication panel in the questions the medicine is not blue, and dosage is missing.
7. Ciproflaxcin should be Ciprofloxacin. Fixed it in the google sheet.
8. Doxycyline should be Doxycycline. Fixed it in the google sheet.
9. Grammar is not correct in the Note. Recommend should be recommended. Maybe drop comma at line end.
![14-medication](https://user-images.githubusercontent.com/3536805/175762219-fdf8b78d-3415-4059-acc3-b1af9835fbf7.png)

### Checked and approved:

1. When next steps display, the required panels display, but they contain typos.

**Comment by anvmn** [comment]
@jeanette-pelizzon 

> 9. Grammar is not correct in the Note. Recommend should be recommended. Maybe drop comma at line end.

Shall we drop the comma?

**Comment by anvmn** [comment]
> 1. Not related to this issue, but I found a typo when clicking around:

Fixed at another PR

> 2. So far I have found no way to indicate some diseases, like gonorrhea or candidiasis at first encounter. I need to finish first encounter, come back, and only then will be the symptom review icon available, where I can specify the symptoms. Is there a way to get these diagnoses at first encounter? What are the preconditions to display symptom review? Panel says patient has noted symptoms that require follow up question. I only noted pregnancy.

Addressed this at another PR. Will need clarifications from @jeanette-pelizzon .

> 3. Currently I cannot reproduce it, but when at subsequent encounter I started with filling out the symptom review I ended up with Next steps not displaying. Just as I was about to finish the encounter, I went back and changed the symptoms, and from that time I am getting the next steps icon.

Mandatory activity to complete is Danger signs. Without completing it, no next steps.

> 4. The Warning popup about gonorrhea is not as descriptive, as in the issue description.

Changed.

> 5. The Warning popup about gonorrhea is not as descriptive, as in the issue description.

Fixed

> 7. Ciproflaxcin should be Ciprofloxacin. Fixed it in the google sheet.
> 8. Doxycyline should be Doxycycline. Fixed it in the google sheet.

Both fixed 

> 9. Grammar is not correct in the Note. Recommend should be recommended
Fixed

**Comment by anvmn** [comment]
> 6. On the medication panel in the questions the medicine is not blue, and dosage is missing.

This is the standard we display for any medication. The recommendation is blue with dosage.
At question it's gray, and dosage is not mandatory. @jeanette-pelizzon please approve it's ok.

**Comment by jeanette-pelizzon** [comment]
- Yes fix comma issue 
- You are correct on the blue / grey medication rules @anvmn 

// @anvmn @balagan73

**Comment by anvmn** [comment]
J I m up g hmm nkkb
